bin/main.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require_relative '../lib/player'
+
+class Tictactoe
+  attr_accessor :board, :name
+
+  def initialize
+    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+    @name = []
+  end
+
+  def display_board
+    puts " #{@board[0]} | #{@board[1]} | #{@board[2]} "
+    puts '-----------'
+    puts " #{@board[3]} | #{@board[4]} | #{@board[5]} "
+    puts '-----------'
+    puts " #{@board[6]} | #{@board[7]} | #{@board[8]} "
+    puts ' '
+  end
+
+  def move_board(idx)
+    @board[idx] = 'X'
+  end
+
+  def input_to_index(user_input)
+    user_input.to_i - 1
+  end
+
+  def turn
+    puts 'Choose a spot between 1-9'
+    spot = gets.strip
```

Comment:
#### `[REQUIRED]` game informs player if selected move is invalid

Why can't we use this variabe `spot` to check for valid moves? :thinking: 

```suggestion
    spot = gets.strip.to_i
    until spot.positive?
      puts 'Please enter a valid move'
      spot = gets.strip.to_i
    end
```